src/EventGrid/EventGrid/Models/PSEventSubscription.cs

```diff
@@ -33,6 +33,7 @@ public PSEventSubscription(EventSubscription eventSubscription)
             this.Topic = eventSubscription.Topic;
             this.EventTtl = eventSubscription.RetryPolicy?.EventTimeToLiveInMinutes;
             this.MaxDeliveryAttempt = eventSubscription.RetryPolicy?.MaxDeliveryAttempts;
+            this.EventDeliverySchema = eventSubscription.EventDeliverySchema;
```

Comment:
is it necessary to have eventSubscription?.EventDeliverySchema

Reply:
Same as above.. 

Not really since eventSubscription is an input parameter and we are not calling this method/class unless we need to display an actual value.